### DISCUSSION TITLE
View the findings from our community survey (June 2024)

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#128571] [Announcements]
We love hearing from our members and we love sharing back what we learned from you from our surveys. Below is an infographic round-up of the key findings and most common feedback from the May-June 2024 Community survey!

Thank you again to every community member who participated! We do look at _every single response_ and use this to inform some of our strategy going forward to make the GitHub Community Discussions the best they can be.

![FY24H2 Survey Infographic (External)](https://github.com/user-attachments/assets/ce6785a1-896b-41ec-a49d-16e5dd2261ce)

### Description
The infographic presents results from the Community Sentiment Survey conducted by community.github.com in June 2024. Key statistics and result commentary includes:

- 7,071 total community solutions
- 27 hour median time to solution
- 38 million page views over the past year

- 66% of respondents say that the space "Meets Expectations" and they have plenty of ideas on how to improve it.

- The majority of community members, 76%, primarily visit the community to learn about new GitHub features or share feedback.

- Regarding how people want to participate, the community is split between industry discussions (22.4%), networking & group projects (22.4%), challenges & hackathons (20.4%), AMAs (18.4%), and off-topic discussions (16.3%).

- Things people like most include being able to help others, warm-hearted users & staff, and that GitHub helps them understand if a problem is known or if GitHub is not acting on it. Verbatim quotes from the survey responses include:
   - "What I like most is being able to help others"
   - "It helps me understand if there is a known solution for a problem, or if the problem is known and Github is [or] is not acting on it"
   - "Warm-hearted users & staff"

- Areas for potential improvement include making the UI/homepage more accessible to users, making it easier to choose the right category names/terms, and frustrations around product feedback feeling unacknowledged by GitHub staff. Verbatim quotes from the survey responses include:
   - "I think I would make it more accessible to users, maybe by creating an icon in the homepage of Ul or github homepage"
   - "Difficult to choose the right category, especially given some have specific names/terms"
   - "There are some pretty important threads here [...] that feel completely ignored by Github staff"

Thank you! 
We plan on regularly running this survey to you so we can continuously improve your community experience. Be on the lookout!

**Comment by Cabdaledaa** [comment]
👋

**Reply by Jimcaribo** [reply]
> 👋

**Comment by opus2024** [comment]
> We love hearing from our members and we love sharing back what we learned from you from our surveys. Below is an infographic round-up of the key findings and most common feedback from the May-June 2024 Community survey!
> 
> 
> 
> Thank you again to every community member who participated! We do look at _every single response_ and use this to inform some of our strategy going forward to make the GitHub Community Discussions the best they can be.
> 
> 
> 
> ![FY24H2 Survey Infographic (External)](https://github.com/user-attachments/assets/ce6785a1-896b-41ec-a49d-16e5dd2261ce)
> 
> 
> 
> ## GitHub Community Sentiment Survey June 2024 Edition 
> 
> 
> 
> ### Description
> 
> The infographic presents results from the Community Sentiment Survey conducted by community.github.com in June 2024. Key statistics and result commentary includes:
> 
> 
> 
> - 7,071 total community solutions
> 
> - 27 hour median time to solution
> 
> - 38 million page views over the past year
> 
> 
> 
> - 66% of respondents say that the space "Meets Expectations" and they have plenty of ideas on how to improve it.
> 
> 
> 
> - The majority of community members, 76%, primarily visit the community to learn about new GitHub features or share feedback.
> 
> 
> 
> - Regarding how people want to participate, the community is split between industry discussions (22.4%), networking & group projects (22.4%), challenges & hackathons (20.4%), AMAs (18.4%), and off-topic discussions (16.3%).
> 
> 
> 
> - Things people like most include being able to help others, warm-hearted users & staff, and that GitHub helps them understand if a problem is known or if GitHub is not acting on it. Verbatim quotes from the survey responses include:
> 
>    - "What I like most is being able to help others"
> 
>    - "It helps me understand if there is a known solution for a problem, or if the problem is known and Github is [or] is not acting on it"
> 
>    - "Warm-hearted users & staff"
> 
> 
> 
> - Areas for potential improvement include making the UI/homepage more accessible to users, making it easier to choose the right category names/terms, and frustrations around product feedback feeling unacknowledged by GitHub staff. Verbatim quotes from the survey responses include:
> 
>    - "I think I would make it more accessible to users, maybe by creating an icon in the homepage of Ul or github homepage"
> 
>    - "Difficult to choose the right category, especially given some have specific names/terms"
> 
>    - "There are some pretty important threads here [...] that feel completely ignored by Github staff"
> 
> 
> 
> Thank you! 
> 
> We plan on regularly running this survey to you so we can continuously improve your community experience. Be on the lookout!
> 
>

**Comment by SumitBoz** [comment]
good one

**Reply by TonyPontana** [reply]
ここには非常に重要な糸がいくつかあります[22.4%]Githubのスタッフに完全に無視されているように感じる

**Comment by Anytar888** [comment]
![1000002856](https://github.com/user-attachments/assets/b42ddd9a-34a7-4edb-959b-b38c7882bd91)
![1000002857](https://github.com/user-attachments/assets/d0fd99c5-badc-4aa1-809b-bc0931bba07e)
![1000002840](https://github.com/user-attachments/assets/c77647cd-072d-4958-bd58-cf0f937c004d)
![1000002879](https://github.com/user-attachments/assets/e677c600-51c4-459c-821d-45cbbdf59ac6)
![26b6b924-6a1d-4727-adf7-3d7a640394e8-1_all_9523](https://github.com/user-attachments/assets/20ff68f5-7f06-4f92-81eb-f52be9422f62)

**Comment by MasterInQuestion** [comment]
One thing I forgot to write in the survey:
    https://github.com/orgs/community/discussions/140201
    .
    "If there is one thing you could change about the GitHub Community Discussions, what would it be?"
    No more "Sign in to view".

    See also: [ https://github.com/microsoft/vscode/issues/237371#issuecomment-2579268033](https://github.com/microsoft/vscode/issues/237371#issuecomment-2579268033)
